backend/src/res/application.properties

```diff
@@ -0,0 +1,8 @@
+spring.datasource.url=jdbc:mysql://ec2-35-178-33-181.eu-west-2.compute.amazonaws.com:3306/development
```

Comment:
this should not be checked in especially not on a public repo

Reply:
Okay, I'll look into putting it into an environment variable.